src/person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_reader :id
+  attr_accessor :name, :age
```

Comment:
- [ ] [Optional][Highly recommended] It would be better professionally if you move these attributes to the start of the class even before the `initialize` method.